package.json

```
@@ -20,6 +20,7 @@
     "lint": "pnpm lint:audit && pnpm lint:prettier '**/package.json' && pnpm lint:eslint '**/*.ts'",
```

Do we want to add `lint:markdown` to `lint` here?
```suggestion
    "lint": "pnpm lint:audit && pnpm lint:prettier '**/package.json' && pnpm lint:eslint '**/*.ts' && pnpm lint:markdown",
```